cli/dcoscli/metrics.py

```diff
@@ -98,6 +98,8 @@ def _node_summary_json(datapoints):
     summary_datapoints = [
         _get_datapoint(datapoints, 'cpu.total'),
         _get_datapoint(datapoints, 'memory.total'),
+        _get_datapoint(datapoints, 'memory.free'),
```

Comment:
here for consistency, could we have `memory.used`? A few lines below there is `filesystem.capacity.used`. So that'd make datapoint pairs of `used` and `total`.

Reply:
Hey @bamarni I do agree that this would make more sense! Unfortunately `memory.used` isn't available in our stats - we only have `memory.free`.